### ISSUE TITLE
Any plan to support Liquid / Elements?

### ISSUE THREAD

**Issue by bstin** [1200wd/bitcoinlib#143]
The elementsd node daemon works in a very similar manner to bitcoind, so shouldn't be too much work?

**Comment by mccwdev** [comment]
Yes would be nice to add but it's not on my planning yet. I might look at it later, and of course anyone is free to create a pull request.
